nupic/algorithms/anomaly.py

```diff
@@ -21,7 +21,9 @@
 
 """Anomaly-related algorithms."""
 
+import collections
```

Comment:
`collections` is newly imported, but not used. Please run your changes through pylint with nupic pylint configuration and  fix findings related to your changes.